app/services/attempts_api/tracker_events.rb

```diff
@@ -171,21 +171,19 @@ def logout_initiated(success:)
       )
     end
 
+    # Tracks when user enrolls their MFA device.
     # @param [Boolean] success
-    # A user has attempted to enroll the Backup Codes MFA method to their account
-    def mfa_enroll_backup_code(success:)
-      track_event(
-        'mfa-enroll-backup-code',
-        success:,
-      )
-    end
-
-    # @param [Boolean] success
-    # A user has attempted to enroll the TOTP MFA method to their account
-    def mfa_enroll_totp(success:)
+    # @param mfa_device_type [String<'backup_code', 'otp', 'personal_key', 'piv_cac',
+    # 'remember_device', 'totp', 'webauthn', 'webauthn_platform'>]
```

Comment:
just double checking my understanding (i just pulled these from a list somewhere) -- you can't enroll a `personal_key` or `remember_device` for mfa, right?

Reply:
Correct

Reply:
updated in https://github.com/18F/identity-idp/pull/12163/commits/3ccc0e0ddc01a679f2e75fbd9a23f9ab501a1ea6